spec/controllers/users/two_factor_authentication_controller_spec.rb

```diff
@@ -165,23 +165,34 @@ def index
 
     context 'when user is webauthn enabled' do
       before do
-        stub_sign_in_before_2fa(build(:user, :with_webauthn))
+        stub_sign_in_before_2fa(create(:user, :with_webauthn))
 
-        allow_any_instance_of(
-          TwoFactorAuthentication::WebauthnPolicy,
-        ).to receive(:enabled?).and_return(true)
+        # allow_any_instance_of(
+        #   TwoFactorAuthentication::WebauthnPolicy,
+        # ).to receive(:enabled?).and_return(true)
```

Comment:
What's the plan with this?

Reply:
oops, forgot to pull that out

Reply:
i rewrote the test so it doesn't need it

Reply:
fixed!